sycl/test-e2e/InvokeSimd/Regression/nbarrier_basic.cpp

```diff
@@ -56,7 +56,7 @@ ESIMD_INLINE void ESIMD_CALLEE_nbarrier(nd_item<1> *ndi) SYCL_ESIMD_FUNCTION {
     nd_item<1> *ndi) SYCL_ESIMD_FUNCTION;
 
 int main(void) {
-  auto Queue = queue{gpu_selector_v};
```

Comment:
do we really not need the selector anymore?

Reply:
Test works exactly the same with or without it, I tried both ways.